overrides/assets/stylesheets/home.css

```diff
@@ -1,3 +1,5 @@
+@import url('//cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css');

```

Comment:
do we need this? I think the theme already bundles font-awesome (https://squidfunk.github.io/mkdocs-material/reference/icons-emojis)

Reply:
see https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#using-icons-in-templates for how to use this in a template like home.html

Reply:
Okay. I've migrated to use native mkdocs icons.